app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListActivity.kt

```
@@ -37,20 +37,17 @@ class FriendsListActivity : AppCompatActivity() {
     }
 
     private fun initViewModel() {
```

@aman3014 I think it was better before, since now if BuildConfig.DEBUG is false then you will be continuing even if the user is null, which I think is a mistake

---

If BuildConfig.DEBUG is false then user should not be null, otherwise it's a bug.

---

Yes it's a bug but I guess it can happen no? And we shouldn't be executing the rest of the code

---

The point is to not have any bugs on the main. This assert is there to prevent other developers from inadvertently making a mistake and creating the friend list activity without signing in.

---

The current code will just suppress this error in production.

---

If you write assert(something) in Android Studio, it suggests you to convert it to

`if (BuildConfig.DEBUG && something)
    error(something violated)
`

---

Okay sounds good!

---

It's changed!